src/main/java/com/example/capstone_project/service/scheduler/TermSchedulerService.java

```diff
@@ -35,10 +35,13 @@ public class TermSchedulerService {
     @Async
     public void startTerm() throws Exception {
         //START TERM
-        List<Term> terms = termRepository.getListTermNeedToStart(TermCode.NEW, LocalDateTime.now());
+        List<Term> terms = termRepository.findAll();
```

Comment:
Ôi em Giang yêu dấu ơi viết query đi chứ đừng lấy hết từ trong Database ra rồi mới for loop trong java như thế.